src/constants/tables.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
Thanks @AbegaM 

I'd really like this list of strings to be declared as `const` constants, and referred to in all instances of use.  That will avoid the potential for typos such as `users` for `_users`.